task5/thirdtask/thirdtask.go

```diff
@@ -0,0 +1,51 @@
+package thirdtask
+ 
+import (
+    "encoding/json"
+    "errors"
+    "fmt"
+)
+ 
+func multiplyByTwo(k *int) error {
+    *k *= 2
+    return nil
+}
+ 
+func printMoreTen(k int) error {
+    if k < 10 {
+        return errors.New("k must be > 10")
+    }
+    fmt.Println(k)
+    return nil
+}
+ 
+func dejson() error {
+    type jsStruct struct {
+        Data int  `json:"data"`
+        OK   bool `json:"ok"`
+    }
+    in := []byte(`{"data": 13, "ok": true}`)
+    var out *jsStruct
```

Comment:
There should be:
```go
    var out *jsStruct
    if err := json.Unmarshal(in, out); err != nil {
```
or 
```go
    var out jsStruct
    if err := json.Unmarshal(in, &out); err != nil {
```
Usually that's you'd want to get. Anyway, it's an advice, not an error.